src/commands/ExecCommand.js

```diff
@@ -38,6 +38,7 @@ export default class ExecCommand extends Command {
   getOpts(pkg) {
     return {
       cwd: pkg.location,
+      shell: true,
```

Comment:
The integration test isn't evaluating the variables before sending the arguments to lerna.